src/main/java/seedu/edulog/logic/commands/RevenueCommand.java

```diff
@@ -9,22 +9,38 @@
  */
 public class RevenueCommand extends Command {
     public static final String COMMAND_WORD = "revenue";
-    public static final String COMMAND_USAGE = COMMAND_WORD + ": calculate total amount of money earned"
-            + "from students who have paid. Example: revenue";
-    public static final String COMMAND_SUCCESS = "Total revenue is $%d";
+    public static final String COMMAND_USAGE = COMMAND_WORD + " [paid/unpaid]"
+            + ": calculate total amount of money earned/not earned"
+            + "from students who have paid/not paid. Example: revenue paid";
+    public static final String COMMAND_SUCCESS = "Total revenue %s is $%d";
+    public static final String PAID = "paid";
+    public static final String UNPAID = "unpaid";
+
+    private String option;
+
+    public RevenueCommand(String option) {
+        this.option = option;
+    }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        return new CommandResult(String.format(COMMAND_SUCCESS, model.getRevenue()));
+        int result = (option.equals(PAID)) ? model.getPaid() : model.getUnpaid();
```

Comment:
Design-wise, I think this option is possible, but could introduce some unwanted coupling. I would think that the separation into a `RevenuePaid` and `RevenueUnpaid` command would better encapsulate the commands. Alternatively, we could simplify the command to just Revenue, and display both paid and unpaid amounts, to simplify overall code complexity and hence bugs for both us and the user. What do you think?